lazyflow/request/threadPool.py

```diff
@@ -37,15 +38,15 @@ class StopException(Exception):
 class QueueObject:
     __slots__ = ("obj", "exc", "priority")
 
-    def __init__(self, *, obj=None, exc=None, priority=None):
+    def __init__(self, *, obj: Any = None, exc: Exception = None, priority: Optional[List[int]] = None) -> None:
```

Comment:
Maybe use a dataclass for this struct? Seems appropriate.

Reply:
Don't really want to import dataclasses here, little benefit as I need my constructor for validation.